prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -474,7 +474,7 @@ abstract class CoolTest {
     suspend fun pollForStepResult(
         reportId: ReportId,
         taskAction: TaskAction,
-        maxPollSecs: Int = 480,
+        maxPollSecs: Int = 600,
```

Comment:
The end2end_up smoke test for the ELR pipeline topic actually has two translate steps (and batch, send).  The flakiness was the result of some time that batch for the second translate report had not run yet and sometimes it had.